README.md

```diff
@@ -1,4 +1,15 @@
 eos-media
 =========
 
-default media for EOS
\ No newline at end of file
+default media for EOS
+
+Updating the default wallpaper
+------------------------------
+
+As of Endless OS 6, we provide a dark variant of the default wallpaper
+
+1. Add the new wallpapers to `default_files/default_images/`
+2. Update the paths to those wallpapers in `default_files/meson.build`
+3. Add licensing information to `default_files/default_images/LICENSE.tsv`
+
+Since the wallpapers are renamed during the build process, the default wallpaper XML does not need to be updated.
```

Comment:
Strictly speaking they're not renamed but installed twice under different names: once as `usr/share/eos-media/desktop-background-C.jpg` and once as `/usr/share/eos-media/default_images/eos6-mountains.jpg`.

Reply:
```suggestion
Since the default wallpapers are also installed with a standardized default file name during the build process, the dynamic wallpaper XML does not need to be updated.
```